src/app/tracing/dialog/station-properties/station-properties.component.html

```diff
@@ -1,4 +1,17 @@
-<div mat-dialog-content>
+<div
+    mat-dialog-content
+    cdkDrag
+    cdkDragRootElement=".cdk-overlay-pane"
+    cdkDragBoundary=".cdk-overlay-container"
+>
+    <div class="example-handle">
+        <svg cdkDragHandle width="24px" fill="currentColor" viewBox="0 0 24 24">
```

Comment:
The material icon `open_with` looks very similar? Could/Should it be used instead? What do you think?

Reply:
As in the other reply above, I agree.

> I'm debating between using open with, since that's the look I'm going for, or drag handle because that's semantically the correct icon.
Though Angular drag and drop is where I got the SVG used here from, and that looks like the open with icon, so we wouldn't really be breaking convention.